pkg/parser/testdata/proto-task-properties.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "commands": [
+      {
+        "cloneInstruction": {
+        },
+        "name": "clone"
+      }
+    ],
+    "environment": {
+      "CIRRUS_OS": "linux"
+    },
+    "instance": {
+      "@type": "type.googleapis.com/org.cirruslabs.ci.services.cirruscigrpc.ContainerInstance",
+      "cpu": 2,
+      "image": "alpine:latest",
+      "memory": 4096
+    },
+    "metadata": {
+      "properties": {
+        "allow_failures": "false",
+        "custom_bool": "true",
+        "experimental": "false",
+        "indexWithinBuild": "0",
+        "timeout_in": "3600",
+        "trigger_type": "AUTOMATIC"
+      }
+    },
+    "name": "main"
+  }
+]
```

Comment:
Was this fixture was manually generated? 🤔

Reply:
```suggestion
]

```

Reply:
I did generate it from the IntelliJ output. Fixed in 84346b970288ec9c6070efa80d0ee82db5479337